docs/components/overview.mdx

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: Overview
+showTOC: false
+---
+
+# Components Overview
+
+Components allow you to add interactive elements to the messages your app sends. They're accessible, customizable, and easy to use.
```

Comment:
this overview seems to be too specific to messages, it should also mention that components are used in modals